Lib/glyphsLib/builder/features.py

```diff
@@ -89,6 +89,13 @@ def _to_ufo_features(font, ufo=None, generate_GDEF=False, skip_export_glyphs=Non
         if feature.notes:
             lines.append("# notes:")
             lines.extend("# " + line for line in feature.notes.splitlines())
+            feature_name = re.search("^Name: (.+)", feature.notes, flags=re.MULTILINE)
+            if feature_name:
+                name = feature_name.groups()[-1]
+                # Replace special chars backslash and doublequote for AFDKO syntax
+                name = name.replace("\x5c", r"\005c")
```

Comment:
I think this would be more readable, I don't immediately remember that backslash codepoint is U+005C
```suggestion
                name = name.replace("\\", r"\005c")
```

Reply:
Sure!